helm/provider.go

```diff
@@ -74,6 +74,12 @@ func Provider() terraform.ResourceProvider {
 				Default:     "gcr.io/kubernetes-helm/tiller:v2.15.1",
 				Description: "Tiller image to install.",
 			},
+			"connection_timeout": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Default:     60,
+				Description: "Number of seconds Helm will wait to establish a connection to tiller.",
```

Comment:
I think "Number of seconds Helm will wait before timing out a connection to tiller." is more accurate here. This makes it sound like a wait